packages/create-cli/src/actions/dependencies.ts

```diff
@@ -1,7 +1,7 @@
 import * as fs from 'fs'
 import * as path from 'path'
-import detectPackageManager from 'which-pm-runs'
-import execa from 'execa'
+import * as whichpm from 'which-pm'
```

Comment:
This seems to be doing something different than `which-pm-runs`. This detects the installation tool and `which-pm-runs` detects which tool was used to run the code

Reply:
Fixed. I simply created a function `getPackageManager()` that uses the same logic used in `which-pm-runs`.